install/kubernetes/helm/meshery/Chart.yaml

```diff
@@ -39,20 +39,23 @@ appVersion: stable-latest
 
 dependencies:
 - name: mesheryistio
-  version: 0.1.0
+  version: 1.0.0
   condition: mesheryistio.enabled
 - name: mesherylinkerd
-  version: 0.1.0
+  version: 1.0.0
   condition: mesherylinkerd.enabled
 - name: mesheryconsul
-  version: 0.1.0
+  version: 1.0.0
   condition: mesheryconsul.enabled
```

Comment:
@Aisuko thanks for this. It would be great if we can revisit the other charts and update their names to include consistent hyphenation (e.g. meshery-consul)

Reply:
No problems. I will fix that in next coming PR